lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -240,10 +241,11 @@ def credit(amount, payment_method, options = {})
         commit(:credit, post)
       end
 
-      def verify(credit_card, options = {})
+      def verify(payment_method, options = {})
+        amount = (eligible_for_0_auth?(payment_method, options) ? 0 : 100)
```

Comment:
Small syntax nitpick but I don't think you need the `()` wrapped around this evaluation